README.md

```diff
@@ -23,6 +23,8 @@ Rector **upgrades your application** for you, with focus on open-source projects
 composer require --dev rector/rector @dev nikic/php-parser 'dev-master#5900d78 as v3.1.1'
 ```
 
+Do you have old PHP on project? Ok, [it is not problem](/docs/HowUseWithOldPhp.md).
```

Comment:
While we're at it, I'd add note for other dependenies in confllict:
"Do you have old PHP or dependencies in conflict?"

Because some people can have Symfony 3.0 and want to use this. They should find link to *composer project* way here.

Reply:
Thank you, you have right!